packages/venia-ui/lib/components/Button/__stories__/button.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+
+import Button from '../button';
+
+import { storiesOf } from '@storybook/react';
+import { Edit2, Menu } from 'react-feather';
+
+import styleOverrides from './button.overrides.module.css';
+
+const stories = storiesOf('Components/Button', module);
+
+const SIZES = ['large', 'medium', 'small'];
+
+const ICON_DIMENSIONS = { width: '18px', height: '18px' };
+
+const ButtonStory = props => {
+    const { design } = props;
+
+    const clickHandler = () => {
+        console.log('Click');
```

Comment:
Double-checking if including console.log() is common practice/adds value in Storybook.

Reply:
No. Ideally we should be using the [actions add-on](https://storybook.js.org/docs/6.3/react/essentials/actions) but this will have to do for now since we do not have that installed.